app/views/applications/process/personal_information.html.slim

```diff
@@ -72,6 +72,7 @@
         ul
           li was not processed through the money claim online (MCOL) or possession claim online (PCOL) services
           li is not for a search or request for duplicate documents (unless the applicant did not receive the originals or had no fixed address when an order was made)
+          p = link_to 'What to do if the fee is one of these', guide_process_application_path(anchor: 'not-eligible'), target: 'blank'
```

Comment:
The header for this section is 'Check the fee is not:' - that's because it covers fees for search as well as MCOL/ PCOL etc. So should leave as is.